plugins/blockvolume/blockprovider/gluster-virtblock/gluster_virtblock.go

```diff
@@ -99,21 +99,33 @@ func (g *GlusterVirtBlk) CreateBlockVolume(name string, size uint64, hostVolume
 		return nil, err
 	}
 
-	err = utils.ExecuteCommandRun("mkfs.xfs", "-f", blockFileName) //nolint: gosec
-	if err != nil {
-		logger.WithError(err).Errorf("failed to format block file " + blockFileName)
-		return nil, err
+	if blockVolOpts.BlkType != "none" {
```

Comment:
why it is compared with none? can this be compared with an empty string which default none for string?

Reply:
"none" is used to represent that no formatting is required, if its "" then the default is "xfs".

Reply:
sorry, am not aware of no formatting us required use case can you explain?

Reply:
https://kubernetes.io/docs/concepts/storage/persistent-volumes/#volume-mode  This one? VolumeMode can be raw as well, not sure about further deatils.